src/js/components/agencyV2/overview/AgencyOverview.jsx

```diff
@@ -154,35 +154,32 @@ const AgencyOverview = ({
                     {cjBlock}
                 </div>
             </div>
-        );
-    }
-
-    const overview = isLoading ? <LoadingMessage /> : (
-        <>
-            <div className="agency-overview__top">
-                <div className="agency-overview__title">
-                    <h3>
-                        {name}
-                        {name && covidDefCodes.length > 0 &&
```

Comment:
This check for `length > 0` was actually fixing a bug that causes `0` to show up instead of the Covid Tooltip
<img width="1029" alt="Screen Shot 2021-06-07 at 11 32 57 AM" src="https://user-images.githubusercontent.com/7108785/121047649-4b6d5900-c784-11eb-882b-5b8ef08c21e6.png">